include/bluetooth/iso.h

```diff
@@ -79,12 +79,13 @@ struct bt_iso_chan_qos {
 	/** @brief Channel direction
 	 *
 	 *  Possible values: BT_ISO_CHAN_QOS_IN, BT_ISO_CHAN_QOS_OUT or
-	 *  BT_ISO_CHAN_QOS_INOUT.
+	 *  BT_ISO_CHAN_QOS_INOUT. Shall be BT_ISO_CHAN_QOS_OUT for broadcast
+	 *  transmitting, and BT_ISO_CHAN_QOS_IN for broadcast receiver.
 	 */
 	uint8_t				dir;
 	/** Channel interval */
 	uint32_t			interval;
-	/** Channel SCA */
+	/** Channel SCA - Only for CIS */
```

Comment:
Perhaps we should split this into it own struct and then have a union for broadcast and unicast specific parameters.

Reply:
Not a bad idea. 

Do you mean something like:

```
struct bt_iso_chan_qos {
    union {
        struct {
            <CIS fields>
        } cis;

        struct {
            <BIS fields>
        } bis;
    };
};
```